frontend/src/features/env/SwitchEnvFeedbackModal.tsx

```diff
@@ -71,6 +99,8 @@ export const SwitchEnvFeedbackModal = ({
     setShowThanksPage(false)
   }, [onChangeEnv, onClose])
 
+  const isPublicFormPage = window.location.pathname.match(publicFormPath)
```

Comment:
this is throwaway code, so it's a nit:

should we use the `AdminSwitchEnvMessage`/`PublicSwitchEnvMessage` to inject the radio options as props instead of creating it here? Removes the conditionals and improves readability:

```tsx
<Radio.RadioGroup>
  {radioOptions.map(option => 
    ...
  }
  <Radio.OthersWrapper ... />
</Radio.RadioGroup>
```

Reply:
https://github.com/opengovsg/FormSG/pull/5245/commits/4677cfcaf59f9e1676da8bbd23e8a661d1cfc3fd